lang/cs/Org.Apache.REEF.IO.Tests/TestLocalFileSystem.cs

```diff
@@ -136,6 +136,22 @@ public void TestCopy()
             Assert.False(fs.Exists(sourceUri));
         }
 
+        [Fact]
+        public void TestIsDirectory()
+        {
+            var fs = GetFileSystem();
+            var directoryUri = new Uri(Path.Combine(Path.GetTempPath(), TempFileName) + "/");
+            var fakeDirectoryUri = new Uri(Path.Combine(Path.GetTempPath(), "fakeDir") + "/");
+            fs.CreateDirectory(directoryUri);
+            var fileUri = new Uri(directoryUri, "testfile");
+            MakeRemoteTestFile(fs, fileUri);
+            Assert.True(fs.IsDirectory(directoryUri));
+            Assert.False(fs.IsDirectory(fakeDirectoryUri));
+            Assert.False(fs.IsDirectory(fileUri));
+            fs.Delete(fileUri);
+            fs.DeleteDirectory(directoryUri);
```

Comment:
nit: cleanup block to make more readable.

Reply:
cleaned up

---
In reply to: [198553473](https://github.com/apache/reef/pull/1473#discussion_r198553473) [](ancestors = 198553473)